src/lib/tsc-watch.ts

```diff
@@ -125,7 +125,6 @@ function spawnTsc({ maxNodeMem, requestedToListEmittedFiles, signalEmittedFiles
   const nodeArgs = [
     ...((maxNodeMem) ? [`--max_old_space_size=${maxNodeMem}`] : []),
     tscBin,
-    ...((signalEmittedFiles || requestedToListEmittedFiles) ? ['--listEmittedFiles'] : []),
```

Comment:
why was this removed?

Reply:
I moved this to `args-manager.ts` so the order of `--build` vs `--listEmittedFiles` can be tested